imports/ui/helpers/time.coffee

```diff
@@ -0,0 +1,44 @@
+Session.setDefault('now', +new Date)
+
+tickInterval = 1000
+
+tick = ->
+  Session.set('now', Session.get('now') + tickInterval)
+setInterval tick, tickInterval
+
+talkTime = do ->
+  second = 'second'
```

Comment:
Given that these variables all appear only once, how about we just use the string literals?